additional_stats.py

```diff
@@ -0,0 +1,58 @@
+def calc_cost(row):
+    # print(row.columns)
+    if type(row['AllocTRES']) == float:
+        return 0
+    else:
+        alloc_tres = row['AllocTRES'].split(',')
+        n_elements = len(alloc_tres)
+        billing = alloc_tres[0].split('=')[0]
+        if billing == 'billing':
+            n_bills = int(alloc_tres[0].split('=')[1])
+        cost = 0
+        for n in range(1, n_elements):
+            resource = alloc_tres[n].split('=')[0]
+            if resource == 'cpu':
+                # row['PartitionTypeX'] = 'CPU'
+                cost = (5 * n_bills * row['CPUtime (hours)']) / 100
+            elif resource == 'gres/gpu':
+                # row['PartitionTypeX'] = 'GPU'
+                cost = (240 * n_bills * row['CPUtime (hours)']) / 100
```

Comment:
same comment as the previous:
you are iterating over n_elements, but you are not adding up the cost?
Shouldn't be `cost += (240 * n_bills * row['CPUtime (hours)']) / 100`